src/platform/qt/CoreController.cpp

```diff
@@ -853,6 +853,8 @@ void CoreController::clearOverride() {
 }
 
 void CoreController::startVideoLog(const QString& path, bool compression) {
+	Q_UNUSED(compression)
```

Comment:
I get the feeling that this parameter isn't supposed to be unused. I should probably look into why instead of squelching the warning.